ethcore/src/miner/miner.rs

```diff
@@ -851,6 +851,37 @@ impl miner::MinerService for Miner {
 		self.transaction_queue.all_transactions()
 	}
 
+	fn pending_transactions_hashes<C>(&self, chain: &C) -> BTreeSet<H256> where
```

Comment:
I think it should be `pending_transaction_hashes` (no double plural `s`), see e.g. [this](https://english.stackexchange.com/questions/203985/hidden-layer-sizes-vs-hidden-layers-sizes).

(I am not a native speaker so take it with a grain of salt)

Reply:
I checked and we're having both usages. This probably should be unified to a single one in the future:

* In `eth_filter` we have `pending_transactions_hashes`.
* In `ethcore`'s Block's views and `encoded`, we have `transaction_hashes`.